### PR TITLE
qt 6.1.0

### PR DESCRIPTION
**Pull request by paperchalice**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Additional libraries are bundled in the tarball, I'm not sure `qt` will do so in future versions.
Add `brotli` for some network feature and `webengine` in future.
[About `FIXME`](https://bugreports.qt.io/browse/QTBUG-89559)
It might be better to put plugins in `lib/plugins/<version>`

### CODE REVIEW

File: Formula/qt.rb [hunk 1]
```diff
@@ -49,29 +49,19 @@ class Qt < Formula
...
 
     config_args = %W[
       -release
 
       -prefix #{HOMEBREW_PREFIX}
       -extprefix #{prefix}
+      -sysroot #{MacOS.sdk_path}
 
       -libexecdir share/qt/libexec
       -plugindir share/qt/plugins
```
Comment (paperchalice) [inline Formula/qt.rb:67]: `lib/plugins/<version>` would be better.

**Comment by paperchalice** [comment]
How to apply patch to subfolder?

**Comment by carlocab** [comment]
See patches in `boost` formula

**Comment by carlocab** [comment]
Is this ready to merge, @paperchalice? LGTM

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**